tools/emails.py

```diff
@@ -5,18 +5,18 @@
 from email.mime.text import MIMEText
 
 
-def send_email(regression, jobs, message, addition=None, deletion=None, admin=False, results=False):
+def send_email(regression, jobs, date, message, change=None, admin=False, results=False):
```

Comment:
We should read the email host, username and password from a config file (or the database).